procedure requests/README.md

```diff
@@ -0,0 +1,134 @@
+# Procedure Request
+
+## XML HL7 > JSON FHIR
+
+A Procedure Request is mapped primarily from a PlanStatement.
+
+| Mapped to (JSON FHIR Document Reference field) | Mapped from (XML HL7 / other source)                                                                                                             |
+|------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| meta.profile                                   | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-ProcedureRequest-1"`                                                |
+| identifier\[0].system                          | `"https://PSSAdaptor/{{losingOdsCode}}"` - where the `{{losingOdsCode}}` is the ODS code of the losing practice                                  |
+| identifier\[0].value                           | ` PlanStatement / id \[@root] `                                                                                                                  |
+| id                                             | ` PlanStatement / id \[@root] `                                                                                                                  |
+| status                                         | fixed value = `"active"`                                                                                                                         |
+| intent                                         | fixed value = `"plan"`                                                                                                                           |
+| authoredOn                                     | ` PlanStatement / availabilityTime \[@value] ` or ` EhrComposition / availabilityTime \[@value] ` or ` EhrExtract / availabilityTime \[@value] ` |
+| occurrenceDateTime                             | ` PlanStatement / effectiveTime / center \[@value] `                                                                                             |
+| subject                                        | reference to the mapped [Patient](../patient/README.md)                                                                                          |
+| note                                           | ` PlanStatement / text `                                                                                                                         |
+| requester.agent                                | ` PlanStatement / participant `                                                                                                                  |
+| code                                           | ` PlanStatement / code `                                                                                                                         |
```

Comment:
if the code element doesn't have a SNOMED code either as the main code or a translation the Adaptor inserts `Transfer-degraded plan` (196451000000104)